### PR TITLE
Localized GetThingsDone

### PR DESCRIPTION
**Pull request by kiridix**

### CODE REVIEW

**Comment by llaske** [comment]
locale.ini is missing. Do you forget it in your commit ?
Plus, you have to declare ln10 as 2nd parameter of the define. 
Currently, there is an error message in the browser console.

**Comment by llaske** [comment]
Unfortunately there is still a bug. The l10n parameter is declared at the wrong place.
I suggest you to use the browser debugger console. On Chrome press Ctrl+Shift+J (Windows / Linux) or Cmd+Opt+J (Mac).
Plus it's important to adapt the locale to the Sugarizer language settings. See for example what is done here: https://github.com/llaske/sugarizer/blob/master/activities/Clock.activity/js/loader.js#L15

**Comment by kiridix** [comment]
There is a bug that l10n can´t read locale.ini

File: activities/GetThingsDone.activity/js/activity.js [hunk 1]
```diff
@@ -7,6 +7,7 @@ define(["sugar-web/activity/activity","sugar-web/datastore","activity/model","ac
 
         // Initialize the activity.
         activity.setup();
+		document.ElementById("new-todo") = l10n.get("new-todo");
```
Comment (tradzik) [inline activities/GetThingsDone.activity/js/activity.js]: Incorrect tabulation.

File: activities/GetThingsDone.activity/index.html [hunk 2]
```diff
@@ -4,6 +4,7 @@
     <meta charset="utf-8" />
     <title>Get Things Done Activity</title>
     <meta name="viewport" content="user-scalable=no, initial-scale=1, maximum-scale=1, minimum-scale=1, width=device-width"/>
+	<link rel="prefetch" type="application/l10n" href="locale.ini" />
```
Comment (tradzik) [inline activities/GetThingsDone.activity/index.html]: Incorrect tabulation.

File: activities/GetThingsDone.activity/js/activity.js [hunk 3]
```diff
@@ -1,4 +1,4 @@
-define(["sugar-web/activity/activity","sugar-web/datastore","activity/model","activity/view","activity/controller"], function (activity,datastore,model,view,controller) {
+define(["sugar-web/activity/activity", "sugar-web/datastore", "activity/model", "activity/view", "activity/controller"], "l10n", function (activity, webL10n, datastore, model, view, controller) {
```
Comment (tradzik) [inline activities/GetThingsDone.activity/js/activity.js]: We don't use spaces in define clauses.

Comment (tradzik) [inline activities/GetThingsDone.activity/js/activity.js]: Changing this line is not needed - we don't have to import `webL10n`, we have already declared global translation interface in `js/loader.js`.

File: activities/GetThingsDone.activity/js/activity.js (hunk 1 above)
Comment (tradzik) [inline activities/GetThingsDone.activity/js/activity.js]: We use our global interface - should be `l10n_s.get("blabla")` instead.

File: activities/GetThingsDone.activity/js/activity.js (hunk 3 above)
Comment (tradzik) [inline activities/GetThingsDone.activity/js/activity.js]: BTW The module name is `webL10n`, not `L10n`. Additionally, you haven't put into the array.

File: activities/GetThingsDone.activity/locale.ini [hunk 4]
```diff
@@ -0,0 +1,6 @@
+[es]
+new-todo=¿Qué más se debe hacer?
+[en]
```
Comment (tradzik) [inline activities/GetThingsDone.activity/locale.ini]: We use new line before every new language declaration.

File: activities/GetThingsDone.activity/locale.ini [hunk 5]
```diff
@@ -0,0 +1,6 @@
...
+new-todo=What else needs to be done?
+[fr]
```
Comment (tradzik) [inline activities/GetThingsDone.activity/locale.ini]: We use new line before every new language declaration.

**Comment by llaske** [comment]
There is still a JavaScript error here https://github.com/llaske/sugarizer/pull/91/files#diff-dfd8ded96c059b5ff29b10a910b9c80fR10

**Comment by kiridix** [comment]
Ive corrected the JS error, there are no errors left at the console :D, tell me if I have to do something more

**Comment by llaske** [comment]
No there is still an error in the console here https://github.com/llaske/sugarizer/pull/91/files#diff-2e7d59fafdfd04e74f0bacea342d2326R10

**Comment by kiridix** [comment]
Ive solved the error, but there is other error with the Items, Im trying to solve it too

**Pull request merged**